docs/project/repository-guide.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Repository guide
+description: A quick guide to the most important and most frequently used IPFS repositories.
+---
+
+# IPFS repository guide
+
+IPFS is a _big_ open-source project, and with that comes a lot of code — and a lot of issue-based discussion that goes with it. The IPFS project uses GitHub both for code development and for road mapping and operations discussions, meaning that both types of activities can happen asynchronously, in the open, and from anywhere on the planet.
+
+If you're looking for specific IPFS-related code, or want to find where to join in a particular discussion, start with this high-level guide to the most prominent and/or frequently used GitHub organizations and repositories in the IPFS project. It's not an exhaustive list of all IPFS organizations and repositories, but instead is intended to help guide you based on what you are trying to do in the following areas:
+
+- [Fetching or contributing to IPFS code](#fetch-or-contribute-to-ipfs-code)
+- [Participating in IPFS project operations](#participate-in-ipfs-project-operations)
+- [Discussing ideas for using IPFS](#discuss-ideas-for-using-ipfs)
+
+## Fetch or contribute to IPFS code
+
+Organization and repository links for the top-level projects shipped as part of the IPFS project and the main repos for each ingredient in the stack upon which IPFS is built.
+
+### Protocol implementations
+
+- [go-ipfs](https://github.com/ipfs/go-ipfs): The reference implementation written in Go.
+- [js-ipfs](https://github.com/ipfs/js-ipfs): The JavaScript implementation of IPFS.
+- [rust-ipfs](https://github.com/rs-ipfs/rust-ipfs): Alpha implementation in Rust.
+
```

Comment:
can you add a link after this list to https://github.com/ipfs/ipfs#protocol-implementations for the full list